airbyte-ci/connectors/base_images/README.md

```diff
@@ -42,7 +42,7 @@ RUN mkdir -p 755 /usr/share/nltk_data
 ### Example for `airbyte/java-connector-base`:
 ```dockerfile
 FROM docker.io/amazoncorretto:21-al2023@sha256:5454cb606e803fce56861fdbc9eab365eaa2ab4f357ceb8c1d56f4f8c8a7bc33
-RUN sh -c set -o xtrace && yum update -y --security && yum install -y tar openssl findutils && yum clean all
+RUN sh -c set -o xtrace && yum update -y --security && yum install -y /usr/sbin/adduser tar openssl findutils && yum clean all && adduser --base-dir /airbyte --uid 1000 --user-group --system airbyte && mkdir --mode 755 /airbyte && mkdir --mode 755 /custom_cache && chown -R airbyte:airbyte /airbyte
```

Comment:
We should align on how we create this use between here and the [platform java base image](https://github.com/airbytehq/airbyte-platform-internal/blob/05b8dbab3cd61665e8e4ac7ca3b9158666821f7e/oss/docker/airbyte-base-java-image/Dockerfile#L7-L20)
```
RUN <<EOF
yum install -y tar shadow-utils
yum update -y --security
yum clean all
rm -rf /var/cache/yum
groupadd --gid 1000 airbyte
useradd --uid 1000 --gid airbyte --shell /bin/bash --create-home airbyte
# Create mount point for secrets
mkdir /secrets
chown -R airbyte:airbyte /secrets
chown -R airbyte:airbyte /app
chown -R airbyte:airbyte /usr/share/pki/ca-trust-source
chown -R airbyte:airbyte /etc/pki/ca-trust
EOF
```

Reply:
@colesnodgrass 👍 I'll chown the same paths.
For the `useradd` / `groupadd` commands: amazon coretto does not bundle these commands, can I yum install them like I did for `adduser`?

Reply:
these should become available upon installing the shadow-utils package